doc/user_guide/user_guide_107.md

```diff
@@ -0,0 +1,143 @@
+# Exasol Power BI Connector 1.0.7 Release - User Guide
+
+## New features
```

Comment:
Usually we don't include the new features here but in the changes_x.x.x.mde file.